### PR TITLE
Add Lastodash sample application to the gallery

### PR DESCRIPTION
**Pull request by matthewchan15**
Added the Lastodash sample application to the dash gallery.

Original repo: https://github.com/n-riesco/lastodash
DDS modified repo: https://github.com/plotly/dash-lastodash
DDS application: https://dash-gallery.plotly.host/dash-lastodash

### CODE REVIEW

**Comment by xhlulu** [comment]
The github link doesn't exist @matthewchan15

**Comment by matthewchan15** [comment]
Need to add fix to this issue: [257](https://github.com/plotly/dash-docs/issues/257)

**Comment by xhlulu** [comment]
don't do this in this PR, please do it in a different PR

**Comment by matthewchan15** [comment]
@xhlulu sorry, mean't to comment this in https://github.com/plotly/dash-docs/pull/266

File: tutorial/gallery.py [hunk 1]
```diff
@@ -238,9 +238,25 @@ def AppSection(app_name,
...
-        )
+        ),
     ]),
 
+    reusable.Row([
+        AppSection(
+            app_name='LAStoDash',
+            app_link='https://dash-gallery.plotly.host/dash-lastodash',
+            code_link='https://github.com/n-riesco/lastodash',
+            img_src='https://raw.githubusercontent.com/plotly/dash-lastodash/master/alcor1.gif?token=Af6ihbal0uF5h5s1C2N8eo2SZFulAMJpks5b71HSwA%3D%3D',
```
Comment (xhlulu) [inline tutorial/gallery.py]: I think you can put directly: https://raw.githubusercontent.com/plotly/dash-docs/master/images/dash-lastodash.gif

It should work once it's merged.

Comment (xhlulu) [reply tutorial/gallery.py]: I think this way is preferred since dash-lastodash repo isn't public, and might be modified in the future

Comment (matthewchan15) [reply tutorial/gallery.py]: @xhlulu approve the review on github, than I can merge it please!

**Comment by xhlulu** [comment]
What comment did you delete?

**Comment by matthewchan15** [comment]
@xhlulu I deleted irrelevant comments in this repo, about addressing https://github.com/plotly/dash-docs/issues/257, which was fixed in another PR.

File: tutorial/gallery.py (hunk 1 above)
Comment (chriddyp) [reply tutorial/gallery.py]: We should move images to the `assets/` folder so that they aren't dependent on 3rd party repos.

Comment (matthewchan15) [reply tutorial/gallery.py]: @chriddyp I will make a PR with all the images for `dash-gallery` being loaded from the assets folder.

**Pull request merged**